eng/Versions.props

```diff
@@ -16,7 +16,8 @@
 
   <PropertyGroup>
     <VersionPrefix>3.1.0</VersionPrefix>
-    <PreReleaseVersionLabel>rc1</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
```

Comment:
@deepchoudhery - You should update the PreReleaseVersionLabel to Preview2 and remove the PreReleaseVersionIteration